go-fuzz-build/cover.go

```diff
@@ -33,6 +30,26 @@ func instrument(pkg, fullName string, fset *token.FileSet, parsedFile *ast.File,
 	}
 	if sonar == nil {
 		file.addImport("go-fuzz-dep", fuzzdepPkg, "CoverTab")
+		for _, element := range info.Types {
```

Comment:
Can we add a comment above this line to elaborate the reason we add import paths here?

Reply:
Sure